src/components/ErrorSuppression/ErrorSuppressionFilters/ErrorSuppressionFilters.js

```diff
@@ -69,21 +69,21 @@ export const getLocations = (facilityValue, checkResultObj) => {
     // TODO: 5609 this might fail
 
     let availLoc = locations?.map((l) => ({
-      id: l.id,
-      label: l.unitId,
+      id: l?.id,
+      label: l?.unitId,
       selected: false,
       enabled: true,
     }));
     if (checkResultObj.locationTypeCode === "LOC") {
       const availStackPipe = locations?.map((l) => ({
-        id: l.id,
-        label: l.stackPipeId,
+        id: l?.id,
+        label: l?.stackPipeId,
         selected: false,
         enabled: true,
       }));
       availLoc = [...availLoc, ...availStackPipe];
     }
-    const locName = availLoc.map((l) => l.label);
+    const locName = availLoc.map((l) => l?.label);
```

Comment:
This is unnecessary, `availLoc` is either `undefined`, an empty array, or an array of objects (never an array of `undefined`).
```suggestion
    const locName = availLoc.map((l) => l.label);
```